src/api/types/generated/fragments/situations.ts

```diff
@@ -0,0 +1,9 @@
+import * as Types from '@atb/api/types/generated/journey_planner_v3_types';
+
+export type SituationFragment = {
+  id: string;
+  situationNumber?: string;
+  reportType?: Types.ReportType;
+  summary: Array<{language?: string; value: string}>;
+  description: Array<{language?: string; value: string}>;
+};
```

Comment:
Nitpick: Should the file be called `situation.ts` instead of plural `situations.ts`?

Reply:
Might be better with singular, but it is also used plural in bff so I don't change this now.